packages/core/.env.example

```
@@ -0,0 +1,18 @@
+
+# Optional. Possible values are development | production. Node environment runtime.
+NODE_ENV=""
+
+# Optional. Disables the injection of third party scripts declared customizations. If not present default is false.
+DISABLE_3P_SCRIPTS=false
+
+# Optional.
+BASE_SITE_URL=''
+
+# OPTIONAL. Way to mock CMS rewsponse in order to test specific response.
+CMS_DATA=''
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)